addons/stock/static/src/js/widgets.js

```diff
@@ -254,11 +254,15 @@ function openerp_picking_widgets(instance){
                 return false;
             });
             this.$('.js_qty').focus(function(){
+                oldjsqtyvalue = parseFloat($(this).val());
                 self.getParent().barcode_scanner.disconnect();
             });
             this.$('.js_qty').blur(function(){
                 var op_id = $(this).parents("[data-id]:first").data('id');
                 var value = parseFloat($(this).val());
+                if (value==oldjsqtyvalue){
+                    return true;
+                }
```

Comment:
by doing it this way, you forget to reactivate the barcode_scanner (3 lines below), why not just change the condition below from "if (value>=0){ self.getParent().set_operation_quantity...}" to "if (value != oldjsqtyvalue and value >= 0) { self.getParent().set_operation_quantity...}" ?